src/components/MenuItem.js

```diff
@@ -42,7 +42,7 @@ const defaultProps = {
     onPress: () => {},
     interactive: true,
     fallbackIcon: Expensicons.FallbackAvatar,
-    brickRoadIndicator: undefined,
+    brickRoadIndicator: null,
```

Comment:
Should this be an `''` as we said in `propTypes` that it would be `'error'` or `''`?

Reply:
Ah yep, good catch.